src/components/visual-tooltip/VisualTooltip.scss

```diff
@@ -0,0 +1,45 @@
+@import '../../styles/variables';
+
+.VisualTooltip {
+    width: 320px;
+    max-width: 320px;
+    padding: 0;
+    font-size: 14px;
+    text-align: center;
+    border-radius: 6px;
+
+    & .tooltip-close-button {
+        top: 16px;
+        right: 16px;
+        width: 16px;
+    }
+
+    &.with-close-button {
+        padding-right: 0;
```

Comment:
Is this still needed if L6 has `padding: 0` already?

Reply:
added a comment, we do need this specific override for specificity.